### PR TITLE
Exercise 1 plus css

### PR DESCRIPTION
**Pull request by GiovanniAlcocer**

### CODE REVIEW

File: exercise 1-semantic and css version.html [hunk 1]
```diff
@@ -49,8 +52,7 @@ <h2>Test this page</h2>
                     </li>
                 </ul>
             </main>
-    
-            <aside style="width: 20%; background-color: lightBlue; margin: 0.3em; padding: 15px;">
+            <aside class="side-content">
                 <h2>HTML</h2>
```
Comment (JaldinDeveloper) [inline exercise 1-semantic and css version.html:58]: Those <p> tags(HTML, CSS, URL) content do not coincide with the content of Nelson's example, but is not a big deal for me... what do you think?

Comment (JaldinDeveloper) [reply exercise 1-semantic and css version.html:58]: also... In my opinion theres two articles inside the section and you wrote only one, what do you think about having two articles dividing "about this page" and "test this page"

File: exercise 1-semantic and css version.html [hunk 2]
```diff
@@ -1,31 +1,34 @@
...
+    <meta name="author" content="Giovanni Alcocer Almanza">
     <base href="https://www.w3schools.com" target="_blank">
     <link rel="icon" href="https://upload.wikimedia.org/wikipedia/commons/4/47/PNG_transparency_demonstration_1.png">
     <title>
         Exercise 1
     </title>
-    <link rel="stylesheet" href="./../FrontEndFirstPractice/mystyle.css">
 </head>
 <body>
     <body>
```
Comment (JaldinDeveloper) [inline exercise 1-semantic and css version.html:17]: double body dude?

File: exercise 1-semantic and css version.html (hunk 1 above)
Comment (GiovanniAlcocer) [reply exercise 1-semantic and css version.html:58]: About the description, i dont think it's a big deal its just a lore ipsum, however i would add the article tag on the "test this page" section

File: exercise 1-semantic and css version.html (hunk 2 above)
Comment (GiovanniAlcocer) [reply exercise 1-semantic and css version.html:17]: fixed

**Pull request merged**